astroquery/ukidss/tests/test_ukidss.py

```diff
@@ -113,15 +113,17 @@ def test_get_images_async_1():
     payload = ukidss.core.Ukidss.get_images_async(
         commons.ICRSCoordGenerator(ra=83.633083, dec=22.0145,
                                    unit=(u.deg, u.deg)),
-        radius=20 * u.arcmin, get_query_payload=True)
+        radius=20 * u.arcmin, get_query_payload=True,
+        programme_id='GPS')
 
     assert 'xsize' not in payload
     assert 'ysize' not in payload
 
     payload = ukidss.core.Ukidss.get_images_async(
         commons.ICRSCoordGenerator(ra=83.633083, dec=22.0145,
```

Comment:
unrelated to this PR, but probably we can now refactor out ``commons.ICRSCoordGenerator`` altogether.